### DISCUSSION TITLE
git上下载下来的示例，本地运行报错，有人遇到吗

### DISCUSSION THREAD

**Discussion by Lisuicheng** [consenlabs/fe-public-template#4] [General]
报错如下：
![1ea1c1cd212753dba6f63fd9127dc45](https://user-images.githubusercontent.com/26526603/136914863-232b2852-4043-4f92-b4db-8e8002351434.png)

**Comment by unix** [comment]
你 clone 的是哪个项目，能具体再给一些信息吗？
这个错误一般是 `node_modules` 内 React 版本有两个，或是 Hooks 规则被破坏了。

**Comment by Lisuicheng** [comment]
token-getting-started下载下来的目录为token-getting-started-master




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年10月14日(星期四) 上午10:08
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [consenlabs/fe-public-template] git上下载下来的示例，本地运行报错，有人遇到吗 (Discussion #4)





 
你 clone 的是哪个项目，能具体再给一些信息吗？
 这个错误一般是 node_modules 内 React 版本有两个，或是 Hooks 规则被破坏了。
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by unix** [reply]
我重新 clone 了一次可以正常运行，然后再次更新了依赖并指定了版本，你可以再试试以下操作：

1. `git clone git@github.com:consenlabs/token-getting-started.git`
2. `cd token-getting-started && cd react`
3. `yarn`
4. `yarn dev`

我猜测可能是本地存在一个 React 版本或是 `node_modules` 是 `npm` 安装导致没有锁定版本，同时出现 16 / 17 报错。
